libseccomp-sys/build.rs

```diff
@@ -5,26 +5,234 @@
 
 use std::env;
 
+#[allow(dead_code)]
+mod versions {
+    include!("versions.rs");
+}
```

Comment:
If we use a versions mod, we can

```suggestion
mod versions;
```

Reply:
I really thought I tried this and it didn't work inside a build script, but I just checked again and it does. Maybe I was tired and wrote "version.rs" instead of versions, or something like that. Will fix.